blogs/2025/05/27/ai-and-remote.md

```diff
@@ -0,0 +1,133 @@
+---
+Order: 100
+TOCTitle: Enhance productivity with AI + Remote Dev
+PageTitle: "Enhance productivity with AI + Remote Dev"
+MetaDescription: Enhance your developer productivity with AI and Remote Development.
+MetaSocialImage: TBD.png
```

Comment:
@olguzzar I'd love to get your input on a social image, maybe grabbing from the video content you're creating!